docs/database-functionalities/triggers.md

```diff
@@ -274,7 +274,14 @@ If the element contains information about a removed vertex label, it's in the fo
   "label": label
 }
 ```
+### Owner
 
+The user who executes the create query are going to be owner of the trigger.
+The privileges of the owner is used when executing `openCypherStatements`,
+in other words the execution of the statements will fail if the
+owner doesn't have the required privileges. More information about how the
+owner affects the trigger can be found in the
+[reference guide](reference-guide/security.md#owners).
```

Comment:
I would signify that this makes sense only in the enterprise version so that the user doesn't get confused if he is using community

Reply:
I will try to emphasize that for both streams and triggers.